mazda/callbacks.h

```diff
@@ -94,10 +124,12 @@ enum class VIDEO_FOCUS_REQUESTOR : u_int8_t {
 class VideoManagerClient : public com::jci::bucpsa_proxy,
                            public DBus::ObjectProxy {
     bool allowedToGetFocus = true;
+    bool inActiveCall = false;
     bool waitsForFocus = false;
 
     MazdaEventCallbacks& callbacks;
     NativeGUICtrlClient guiClient;
+    BTHFManagerClient bthfMgr;
```

Comment:
moved this to VideoManagerClient  to use the bthfstate signal to keep video focus